index.bs

```diff
@@ -135,7 +135,7 @@ interface VRDisplay : EventTarget {
    * Begin presenting to the VRDisplay. Must be called in response to a user gesture.
    * Repeat calls while already presenting will update the VRLayer being displayed.
    */
-  Promise&lt;void&gt; requestPresent(VRLayer layer);
+  Promise&lt;void&gt; requestPresent(sequence&lt;VRLayer&gt; layer);
```

Comment:
should we change [`getLayers`](https://github.com/MozVR/webvr-spec/blob/present_array/index.bs#L148) to always be required to be a sequence?